src/Language/Haskell/Brittany/Internal.hs

```diff
@@ -471,7 +468,9 @@ ppModule lmod@(L _loc _m@(HsModule _name _exports _ decls _ _)) = do
     let mBindingConfs =
           declBindingNames <&> \n -> Map.lookup n $ _icd_perBinding inlineConf
     filteredAnns <- mAsk
-      <&> \annMap -> Map.findWithDefault Map.empty declAnnKey annMap
+      <&> \annMap ->
+        Map.union (Map.findWithDefault Map.empty annKey annMap) $
```

Comment:
I couldn't tell you why this is necessary, but it does in fact solve the problem I was having.